### PR TITLE
api link edits

### PR DESCRIPTION
**Pull request by KellieHawks**
api broken link edits, catching links that did not display correctly. 

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)... because...

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>
- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @KellieHawks as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_api/endpoints/messaging/schedule_messages/post_schedule_messages.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ description: "This article outlines details about the Schedule Messages Braze en
 /messages/schedule/create
 {% endapimethod %}
 
-The create schedule endpoint allows you to schedule a Campaign, Canvas, or other message to be sent at a designated time and provides you with an identifier to reference that message for updates. If you are targeting a segment, a record of your request will be stored in the [Developer Console][41] after all scheduled messages have been sent.
+The create schedule endpoint allows you to schedule a Campaign, Canvas, or other message to be sent at a designated time and provides you with an identifier to reference that message for updates. If you are targeting a segment, a record of your request will be stored in the <a href="https://dashboard-01.braze.com/app_settings/developer_console/activitylog/">Developer Console</a> after all scheduled messages have been sent.
```
Comment (zzhaobraze) [inline _docs/_api/endpoints/messaging/schedule_messages/post_schedule_messages.md]: is there a reason for the html vs using regular markdown? ie [Developer Console](https://dashboard-01.braze.com/app_settings/developer_console/activitylog/)

File: _docs/_api/endpoints/messaging/send_messages/post_send_triggered_campaigns.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ description: "This article outlines details about the Send Campaign Messages via
 
 API Triggered Delivery allows you to house message content inside of the Braze dashboard while dictating when a message is sent, and to whom via your API.
 
-The send endpoint allows you to send immediate, ad-hoc messages to designated users. If you are targeting a segment, a record of your request will be stored in the [Developer Console][41].
+The send endpoint allows you to send immediate, ad-hoc messages to designated users. If you are targeting a segment, a record of your request will be stored in the <a href="https://dashboard-01.braze.com/app_settings/developer_console/activitylog/">Developer Console</a>.
```
Comment (zzhaobraze) [inline _docs/_api/endpoints/messaging/send_messages/post_send_triggered_campaigns.md]: Same as above

File: _docs/_api/endpoints/messaging/schedule_messages/post_schedule_messages.md (hunk 1 above)
Comment (KellieHawks) [reply _docs/_api/endpoints/messaging/schedule_messages/post_schedule_messages.md]: The normal markdown was not showing! had to switch to regular markdown to make the link work.

Comment (zzhaobraze) [reply _docs/_api/endpoints/messaging/schedule_messages/post_schedule_messages.md]: oh if this is in the api tag, then the number reference won't work (this is due to the link being outside the plugin when rendering, so it doesn't know the url), but the above should work.

Comment (KellieHawks) [reply _docs/_api/endpoints/messaging/schedule_messages/post_schedule_messages.md]: I see. fixed it.

**Pull request merged**